### End of chess game status message.
#
# This message is published after a game has terminated.
#

Header      header      # standard ROS header
ChessColor  winner      # who won, if any winner
ChessResult result      # how the game ended
